pkg/tasks/snapshot_helper.go

```diff
@@ -97,33 +102,79 @@ func (sh *SnapshotHelper) Run(versionHref string) error {
 	if err != nil {
 		return err
 	}
+	err = sh.payload.SaveSnapshotIdent(snap.UUID)
+	if err != nil {
+		return fmt.Errorf("unable to save snapshot ident: %w", err)
+	}
+	err = sh.payload.SaveDistributionTaskHref(snap.DistributionHref)
+	if err != nil {
+		return fmt.Errorf("unable to save distribution task href: %w", err)
+	}
+
```

Comment:
the distribution href is saved earlier in the function, so I think it's okay to remove this

Reply:
oops, removed 🫡